test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -1744,6 +1745,7 @@ protected Settings nodeSettings(int nodeOrdinal) {
         Settings.Builder builder = Settings.builder()
             // Default the watermarks to absurdly low to prevent the tests
             // from failing on nodes without enough disk space
+            .put(DestructiveOperations.REQUIRES_NAME_SETTING.getKey(), false)
```

Comment:
Would you mind moving this setting either above the comment or below the `LOW_LEVEL_CANCELLATION_SETTING` line? Right now it's between a comment and the settings that the comment refers to, which isn't ideal.

Reply:
Moved up.